nuxeo-core/nuxeo-core-test/src/main/java/org/nuxeo/ecm/core/test/StorageConfiguration.java

```diff
@@ -177,6 +183,17 @@ protected void clearMongoDB(MongoDBRepositoryDescriptor descriptor) throws Unkno
         }
     }
 
+    protected void initExternal() {
+        // Get DBSHelper by reflection
+        String className = String.format("org.nuxeo.ecm.storage.%s.DBSHelperImpl", coreType);
```

Comment:
The current package for mongodb is org.nuxeo.ecm.core.storage.mongodb. Why not keep the "core" part?